dist/plugins/index.html

```diff
@@ -61,6 +61,55 @@ <h1 id="plugins"><a href="#plugins" class="linkable" style="margin-left:-2rem; m
 Some plugins are maintained in-house, while others are provided by the community.</p>
 <p>On this page, we provide an <code>npm</code> link and total downloads per month for any plugins
 we find or publish ourselves, so you only need to look in one place.</p>
+<!-- 
+
+### [gateway-lambda](gateway-lambda)
+
+[![version][p-npm-version]][p-npm-url]
+[![downloads][p-npm-downloads]][p-npm-url]
+
+[p-npm-version]: https://img.shields.io/npm/v/seneca-gateway-lambda.svg?style=flat-square
+[p-npm-downloads]: https://img.shields.io/npm/dm/seneca-gateway-lambda.svg?style=flat-square
+[p-npm-url]: https://www.npmjs.com/package/@seneca/gateway-lambda
+
+Handle incoming messages within other frameworks.
+
+### [gateway-auth](gateway-auth)
+
+[![version][p-npm-version]][p-npm-url]
+[![downloads][p-npm-downloads]][p-npm-url]
```

Comment:
let's use the "standard" tags from more recent plugins, like: https://github.com/senecajs/seneca-gateway

Reply:
Included here 8ec7b12, badges are temporarily all seneca-gateway for testing/formatting purposes